packages/headless/src/ssr-commerce.index.ts

```diff
@@ -0,0 +1,222 @@
+
+polyfillCryptoNode();
+
+// 3rd Party Libraries
+export type {Unsubscribe, Middleware} from '@reduxjs/toolkit';
+export {createAction, createAsyncThunk, createReducer} from '@reduxjs/toolkit';
+export type {AnalyticsClientSendEventHook} from 'coveo.analytics';
+export type {Relay} from '@coveo/relay';
+
+// Main App
+export type {
+  CommerceEngineOptions,
+  CommerceEngineConfiguration,
+} from './app/commerce-engine/commerce-engine';
+export type {
+  SSRCommerceEngine as CommerceEngine,
+  CommerceEngineDefinition,
+  CommerceEngineDefinitionOptions,
+} from './app/commerce-engine/commerce-engine.ssr';
+export {defineCommerceEngine} from './app/commerce-engine/commerce-engine.ssr';
+export {getSampleCommerceEngineConfiguration} from './app/commerce-engine/commerce-engine-configuration';
+
+// export type
+export type {CoreEngine, ExternalEngineOptions} from './app/engine';
+export type {
+  EngineConfiguration,
+  AnalyticsConfiguration,
+  AnalyticsRuntimeEnvironment,
+} from './app/engine-configuration';
+export type {
+  ControllerDefinitionWithoutProps,
+  ControllerDefinitionWithProps,
+  ControllerDefinitionsMap,
+  InferControllerFromDefinition,
+  InferControllersMapFromDefinition,
+  InferControllerStaticStateFromController,
+  InferControllerStaticStateMapFromControllers,
+  InferControllerStaticStateMapFromDefinitions,
+} from './app/ssr-engine/types/common';
+export type {Build} from './app/ssr-engine/types/build';
+export type {
+  EngineDefinition,
+  InferStaticState,
+  InferHydratedState,
+  InferBuildResult,
+} from './app/ssr-engine/types/core-engine';
+export type {LoggerOptions} from './app/logger';
+
+export type {LogLevel} from './app/logger';
+
+// State
+export type {
+  SearchParametersState,
+  SearchAppState,
+} from './state/search-app-state';
+
+//#region Controllers
+export type {
+  Controller,
+  Subscribable,
+} from './controllers/controller/headless-controller';
+
+export type {CategoryFacet} from './controllers/commerce/core/facets/category/headless-commerce-category-facet';
+export type {
+  DateFacet,
+  DateFacetState,
+} from './controllers/commerce/core/facets/date/headless-commerce-date-facet';
+export type {RegularFacetValue} from './controllers/commerce/core/facets/headless-core-commerce-facet';
+export type {
+  NumericFacet,
+  NumericFacetState,
+} from './controllers/commerce/core/facets/numeric/headless-commerce-numeric-facet';
+export type {RegularFacet} from './controllers/commerce/core/facets/regular/headless-commerce-regular-facet';
+
+// TODO: KIT-3391 - export other SSR commerce controllers
+
+//#endregion
+
+// Selectors
+export {
+  baseFacetResponseSelector,
+  facetRequestSelector,
+  facetResponseSelector,
+  facetResponseSelectedValuesSelector,
+} from './features/facets/facet-set/facet-set-selectors';
+
+export {
+  currentPageSelector,
+  maxPageSelector,
+  currentPagesSelector,
+} from './features/pagination/pagination-selectors';
```

Comment:
Same comment as above, for now I don't think we should export any selectors unless we're certain they're gonna be useful